NEWS.md

```diff
@@ -1,4 +1,41 @@
-## 5.7.0 - Unreleased
+## 5.8.0 - Unreleased
+
+## 5.7.0 - Released Poppy R2 2023
+The focus of this release was to implement Vertx SFTP client for Batch Voucher Export
```

Comment:
the feature to pay against previous fiscaly year is also important amout of work that should be added here in this description

Reply:
Added